pkg/process/util/netns_windows.go

```diff
@@ -0,0 +1,9 @@
+package util
```

Comment:
Silly question, but do these files need build tags?

Reply:
With `_windows` filename suffix, the compiler should restrict it to windows only. This goes into more detail: https://dave.cheney.net/2013/10/12/how-to-use-conditional-compilation-with-the-go-build-tool